projects/mercury/src/file/OrganisationPage.js

```diff
@@ -0,0 +1,168 @@
+import React, {useContext, useEffect, useState} from 'react';
+import Grid from '@material-ui/core/Grid';
+import {withRouter} from "react-router-dom";
+import queryString from "query-string";
+
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import {Divider, Switch, withStyles} from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+import FileBrowser from "./FileBrowser";
+import {getPathInfoFromParams, splitPathIntoArray} from "./fileUtils";
+import * as consts from '../constants';
+import OrganisationBreadcrumbsContextProvider from "../collections/OrganisationBreadcrumbsContextProvider";
+import {useMultipleSelection} from "./UseSelection";
+import LoadingOverlay from "../common/components/LoadingOverlay";
+import SearchBar from "../search/SearchBar";
+import BreadCrumbs from "../common/components/BreadCrumbs";
+import usePageTitleUpdater from "../common/hooks/UsePageTitleUpdater";
+import styles from "./FilesPage.styles";
+import {getMetadataViewsPath, RESOURCES_VIEW} from "../metadata/views/metadataViewUtils";
+import UserContext from "../users/UserContext";
+import MetadataViewContext from "../metadata/views/MetadataViewContext";
+import type {User} from "../users/UsersAPI";
+import {MetadataViewOptions} from "../metadata/views/MetadataViewAPI";
+import type {Match} from "../types";
+import {handleTextSearchRedirect} from "../search/searchUtils";
+
+type ContextualOrganisationPageProperties = {
+    match: Match;
+    history: History;
+    location: Location;
+    classes: any;
+};
+
+type ParentAwareOrganisationPageProperties = ContextualOrganisationPageProperties & {
+    currentUser: User;
+    openedPath: string;
+    views: MetadataViewOptions[];
+}
+
+type OrganisationPageProperties = ParentAwareOrganisationPageProperties & {
+    isOpenedPathDeleted: boolean;
+};
+
+export const OrganisationPage = (props: OrganisationPageProperties) => {
```

Comment:
Just a suggestion, I would just name this `RootDirectoryPage`, "Organisation" already associates this with some organisation/institute entity (in MicroMAIS model there even was organisation entity above department)